### PR TITLE
Fix: Unimplemented type for TryAddOperator

### PR DESCRIPTION
**Pull request by nickgerrets**
TryAddOperator used in the debugging of bitpacking for type `hugeint_t` was unimplemented, causing an abort when running the debug build. This PR implements the TryAddOperator for `hugeint_t`.

### CODE REVIEW

File: src/function/scalar/operators/add.cpp [hunk 1]
```diff
@@ -160,6 +160,12 @@ bool TryAddOperator::Operation(int64_t left, int64_t right, int64_t &result) {
 	return true;
 }
 
+template <>
+bool TryAddOperator::Operation(hugeint_t left, hugeint_t right, hugeint_t &result) {
+	result = left;
+	return Hugeint::AddInPlace(result, right);
```
Comment (Tmonster) [inline src/function/scalar/operators/add.cpp:166]: If I'm not mistaken, this also changes the value of left correct? Is that the desired behavior?

Comment (nickgerrets) [reply src/function/scalar/operators/add.cpp]: Good catch. `result` gets set (to `left`) even when the `AddInPlace` would overflow, which isn't in line with the behaviour of other `TryAddOperator`'s. I made some changes.

Comment (carlopi) [reply src/function/scalar/operators/add.cpp]: 64bits does change result (potentially leaving garbage there), I think it would have been fine either way.

**Comment by carlopi** [comment]
Code coverage will probably need to updated, given function is now a couple of lines longer.

I would consider adding a test that triggers this behaviour, that might be better than bumping coverage, but not strictly needed.

**Comment by nickgerrets** [comment]
As @carlopi suggested, it is better to actually add a test for the code coverage. Other `TryAddOperator`'s are tested through Scalar functions where `x + x` in a query would trigger the `TryAddOperator`. However this was explicitly restricted for the `HUGEINT` type. I've removed that restriction as that seemed the most logical approach to me.

**Comment by hannes** [comment]
@Tmonster can you have another look please?

**Review by Tmonster — APPROVED** [review]
looks good to me!

**Pull request merged**